shortify/redirect_test.go

```diff
@@ -41,6 +47,23 @@ func (suite *RedirectSuite) TestSaveNewRecord() {
 	assert.NotEmpty(t, redir.Token)
 }
 
+func (suite *RedirectSuite) TestTokenMustBeUnique() {
```

Comment:
To be honest, this won't really happen. Using the endpoint, we first check to see if the URL is there before creating a new one, and the token is based on the id which the DB says will be unique.

This test was mostly just to ensure that the `gorp` unique thing worked.

I'll rename though, your suggestion is clearer